lib/db.js

```diff
@@ -151,10 +153,16 @@ DB.prototype.dropTable = function(domain, bucket) {
     var tableName = domain + '_' + bucket;
     var deleteRequest = function(schema) {
         var queries = [
-            {sql: 'drop view [' + tableName + '_indexView]'},
             {sql: 'drop table [' + tableName + '_data]'},
             dbu.buildDeleteQuery(self.schemaTableName, {'table' : tableName})
         ];
+        var secondaryIndexNames = Object.keys(schema.secondaryIndexes);
+        if (secondaryIndexNames.length > 0) {
```

Comment:
@d00rman It's also controlling line 164 - dropping of the secondary indexes table